associated-token-account/program/src/processor.rs

```diff
@@ -76,7 +76,11 @@ pub fn process_create_associated_token_account(
         &[bump_seed],
     ];
 
-    let account_len = get_account_len(spl_token_mint_info, spl_token_program_info)?;
+    let account_len = get_account_len(
+        spl_token_mint_info,
+        spl_token_program_info,
+        vec![spl_token::extension::ExtensionType::FixedOwner],
```

Comment:
Here we see https://github.com/solana-labs/solana-program-library/pull/2852 being used